### ISSUE TITLE
Is it possible to handle 2 TrackPlayers in the same app?

### ISSUE THREAD

**Issue by Xaninho** [DoubleSymmetry/react-native-track-player#1163]
Sup y'all. I have a very basic music streaming app. The core functionalities of the app are to play an online radio stream and a bunch of playlists. The playlists are working wonders thanks the community discussions and docs. However things get tricky when I add a second Track Player. Let's call them RadioPlayer and PlaylistPlayer.

The project fortunally reads my radio stream url and plays it in my RadioPlayer, awesome. However in the second screen (the PlaylistsPlayer) it's supposed to be a different player. If I open up a music using in the PlaylistsPlayer, suddenly the commands from my RadioPlayer screen now affect the PlaylistsPlayer.

Bottom line, I would like to make them more separate. When one plays, the other shuts down, that's the logic. Is it possible to use 2 TrackPlayers? Or will I have to write some detailed logic using only 1 TrackPlayer?

Thanks in advance, thank you for taking the time to read this!

**Comment by HP161** [comment]
are you fix bug this?

**Comment by KrisLau** [comment]
Did you ever figure this out?
